okhttp-logging-interceptor/src/test/java/okhttp3/logging/HttpLoggingInterceptorTest.java

```diff
@@ -580,6 +578,60 @@ private void bodyGetNoBody(int code) throws IOException {
         .assertNoMoreLogs();
   }
 
+  private RequestBody gzip(final RequestBody body) {
```

Comment:
This appears quite useful, should we add this for OkHttp 5? cc @swankjesse

Reply:
Yeah, though it’d probably be even more useful for us to ship a 1st party interceptor that does it? I think it’d need to be opt-in ’cause actual receiving servers don’t support gzip universally.

Reply:
Where should it live?

Reply:
RequestBody.gzip for now.